e2e-tests/README.md

```diff
@@ -25,11 +25,18 @@ rest of Socorro so you're not mixing requirements:
 	$ # your dev virtualenv.
 	$ pip install -r requirements.txt
 
-An additional constraint for Firefox users, with the release of Firefox 48, Webdriver support is currently broken until GeckoDriver is out of Alpha. We suggest using an older version of Firefox which can be [downloaded here][firefoxdownloads].
+An additional constraint for Firefox users, with the release of Firefox 48, Webdriver support is currently broken until GeckoDriver is out of Alpha. We suggest using an older version of Firefox which can be [downloaded here][firefoxdownloads]. Note that on Linux, you don't need to downgrade your Firefox install in order to use an older version.
 
 If you have multiple versions of Firefox installed, you can specifiy a specific one by using the `--firefox-path <path to firefox binary>` flag.
 
-___Running the tests on stage___
+___Using an older version of Firefox on Linux___
```

Comment:
I think we should remove this section. We should default to testing against Firefox latest when possible.